dockers/docker-fpm-frr/frr/supervisord/critical_processes.j2

```diff
@@ -9,5 +9,4 @@ program:pimd
 program:frrcfgd
 {%- else %}
 program:bgpcfgd
-program:staticroutebfd
```

Comment:
But Isn't this a critical process?

Reply:
staticroutebfd itself supports recovery from restart/crash. So it is not necessary to set it to critical process. When a critical process crashes, the bgp container itself will restart, which has more impact. to reduce impact to the bgp container, removed it from critical process, and let supervisord restart staticroutebfd only (see change in supervisord.conf.j2 below).